decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -239,7 +239,7 @@ def gem_modifier
       def branch
         return if options[:path]
 
-        @branch ||= options[:edge] ? "develop" : options[:branch].presence
+        @branch ||= options[:edge] ? "webpacker-dep-docs" : options[:branch].presence
```

Comment:
⚠️ Should we rollback this to develop before merging this?

Reply:
If we rollback this before tests won't pass. I can create another PR just after this PR being merged